howsmyssl.go

```diff
@@ -136,11 +136,22 @@ func main() {
 	}
 	oa := newOriginAllower(blockedOrigins, hostname, gclog, expvar.NewMap("origins"))
 
+	staticHandler := http.NotFoundHandler()
+	webHandler := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I agree, that would be the ideal solution. The hangup is that handleWeb doesn't implement http.Handler. When not headless, `webHandler` above will contain `handleWeb` which is type `func(http.ResponseWriter, *http.Request)`.

Resolving this would involve a little more of a refactor. Thoughts?